pkgs/tools/misc/h5utils/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchurl, hdf5, lib }:
+
+let
+  mpi = hdf5.mpi;
```

Comment:
(non blocking) This was not helping me to read your code. So from my point of view, this could be removed.